src/components/theme/contrast-colors.js

```diff
@@ -18,14 +18,14 @@ function getYiq(color: string) {
 
 // Lighten and Darken should be absolute
 // https://github.com/Qix-/color/issues/53
```

Comment:
Should this folder be moved under `utils` or `styles`? Since its mostly JS.
Also I think you can convert this to .ts pretty fast.

Reply:
Potentially so.. after talking through some potential solutions around theming other components like Button, I don't think we'll need a custom theme provider anymore for providing "defaults" we can just do something like so:

`
// import styled from 'styled-components'
// import defaultTheme from 'components/theme';

const StyledButton = styled.button(({ theme = defaultTheme, ...props }) => ({
    backgroundColor: theme.primary.background,
    borderColor: theme.primary.border,
}))

const Button = (props) => (
    <StyledButton>
        {text}
    </StyledButton>
)

export default Button;`

Reply:
Moved to utils and styles hybrid